test/roundtrip/multi-value-block-type.txt

```diff
@@ -0,0 +1,149 @@
+;;; TOOL: run-roundtrip
+;;; ARGS: --stdout --reloc
+(module
+  ;; 64 random types to force the block type below to get encoded as sleb
```

Comment:
Why 64?    I guess because once we hit a 7 bit number the signed vs unsigned encoding differ?

Reply:
Yes, that's why. 64 must be encoded as two bytes as an SLEB, but can be encoded in just one byte as LEB.